prow/config.yaml

```diff
@@ -6191,6 +6191,41 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
+- interval: 30m
+  name: ci-kubernetes-e2e-gce-ubuntu-1-7-node-prow
+  spec:
+    containers:
+    - args:
+      - --repo=k8s.io/kubernetes=release-1.7
+      - --root=/go/src
+      - --timeout=70
+      env:
+      - name: GOPATH
+        value: /go
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
```

Comment:
I think change this to `GCE_USER`, and remove the `GCE_USER=jenkins` line in the scenario will fix the public key issue.